drivers/tee/optee/core.c

```diff
@@ -27,6 +36,102 @@
 
 #define OPTEE_SHM_NUM_PRIV_PAGES	CONFIG_OPTEE_SHM_NUM_PRIV_PAGES
 
+#if defined(CONFIG_OPTEE_HV_IKGT)
+#define OPTEE_VMCALL_SMC       0x6F707400
+#endif
+
+#if defined(CONFIG_OPTEE_VSOCK)
+#define VIRTIO_SHM_COPY_REQ 	0x5a5a5a5a
+#define VIRTIO_SMC_BUFFER_LEN	0x40
+#define VIRTIO_VSOCK_BUFF_LEN	0x10000
+#define VIRTIO_VSOCK_TEE_PORT	1234
+
+static struct socket *host_smc_sock = NULL;
+static struct socket *tee_sock = NULL;
+unsigned long optee_shm_offset = 0;
+
+static DEFINE_SEMAPHORE(optee_smc_lock);
+#endif
+
+#if defined(CONFIG_OPTEE_IVSHMEM)
+#define PCI_DRV_NAME	"optee-ivshmem"
+
+#define IVPOSITION_OFF	0x08
+#define DOORBELL_OFF	0x0C
+
+#define OPTEE_SHM_QUEUE_SIZE	64
+
+#define OPTEE_HANDLE_DONE		0xa5a5a5a5
```

Comment:
Suggest to keep same indent as line 62 (some tab) for line 64 and 66 (two tabs)

Reply:
Will change it.